docs/features/LOGGING.md

```diff
@@ -1,13 +1,15 @@
 # Logging
 
+## NOTE: The logging features outlined in this document are to be included in version 0.3.0 and newer builds of Open Crawler.
```

Comment:
Rather than a title, this would look better as an `!IMPORTANT` block.
Something like:

```
> [!IMPORTANT]
> The logging features outlined in this document are to be included in version 0.3.0 and newer builds of Open Crawler.
```